### ISSUE TITLE
Crash on link with empty `href` attribute.

### ISSUE THREAD

**Issue by nok-ko** [xavdid/epub-wordcount#4]
```$ node cli.js ~/Calibre\ Library/D.\ D.\ Webb/The\ Gods\ are\ Bastards\ \(19\)/The\ Gods\ are\ Bastards\ -\ D.\ D.\ Webb.epub
/Users/stan/code/bastards/epub-wordcount/node_modules/epub/epub.js:649
                link = path.concat([(linkparts.shift() || "")]).join("/").trim(),
                                               ^

TypeError: linkparts.shift is not a function
    at EPub.<anonymous> (epub-wordcount/node_modules/epub/epub.js:649:48)
    at String.replace (<anonymous>)
    at EPub.<anonymous> (epub-wordcount/node_modules/epub/epub.js:647:19)
    at EPub.<anonymous> (epub-wordcount/node_modules/epub/epub.js:702:13)
    at epub-wordcount/node_modules/epub/epub.js:24:20
    at epub-wordcount/node_modules/adm-zip/zipEntry.js:83:43
    at InflateRaw.<anonymous> (epub-wordcount/node_modules/adm-zip/methods/inflater.js:24:21)
    at InflateRaw.emit (events.js:187:15)
    at endReadableNT (_stream_readable.js:1094:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```
I generated the ePub I'm converting, and I happen to know that all of the links in it are empty (`<a href="">Link</a>`). This issue is fixed in other versions of `epub.js`: https://github.com/julien-c/epub/pull/41/files

**Comment by xavdid** [comment]
Good call! i'm doing some refactoring and will see if I can cleanly update my fork (or incorporate their bug fix). I'll hopefully have that done by the end of the week.

**Comment by xavdid** [comment]
Fixed in aaf3cbd7bad80d49adb845aa0673b717440a02dc and released as part of `v2.0.0`. Thanks for reporting!

**Comment by nok-ko** [comment]
Glad to have helped.
